src/molluscpalette.cpp

```diff
@@ -0,0 +1,100 @@
+#include "molluscpalette.hpp"
+
+#include <fstream>
+#include <array>
+#include <iostream>
+
+#include <QString>
+#include <QVector3D>
+
+MolluscPalette::MolluscPalette(QString dataPath)
+{
+    this->loadData(dataPath);
+    this->fillBuckets();
+}
+
+MolluscPalette::~MolluscPalette()
```

Comment:
is there a reason why you're providing an empty destructor instead of using the implicit one?